src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return (basePrice -= LONG_TERM_DISCOUNT);
```

Comment:
There's no need to use the assignment operator (`-=`) inside the return statement. Simply subtract the discount from the base price and return the result. Using `-=`, in this case, is misleading because it implies that the variable `basePrice` will be used after its value has changed, which is not the case.